documentation/asciidoc/computers/processors/bcm2711.adoc

```diff
@@ -1,18 +1,18 @@
 == BCM2711
 
-This is the Broadcom chip used in the Raspberry Pi 4 Model B, the Raspberry Pi 400, and the Raspberry Pi Compute Module 4. The architecture of the BCM2711 is a considerable upgrade on that used by the SoCs in earlier Raspberry Pi models. It continues the quad-core CPU design of the BCM2837, but uses the more powerful ARM A72 core. It has a greatly improved GPU feature set with much faster input/output, due to the incorporation of a PCIe link that connects the USB 2 and USB 3 ports, and a natively attached Ethernet controller. It is also capable of addressing more memory than the SoCs used before.
+This is the Broadcom chip used in the Raspberry Pi 4 Model B, the Raspberry Pi 400, and the Raspberry Pi Compute Module 4. The architecture of the BCM2711 is a considerable upgrade on that used by the SoCs in earlier Raspberry Pi models. It continues the quad-core CPU design of the BCM2837, but uses the more powerful ARM A72 core. It has a greatly improved GPU feature, and much faster input/output due to the incorporation of a PCIe link that connects the USB ports, as well as a natively attached Ethernet controller. It is also capable of addressing more memory than previous chips.
 
-The ARM cores are capable of running at up to 1.5 GHz, making the Raspberry Pi 4 about 50% faster than the Raspberry Pi 3B+. The new VideoCore VI 3D unit now runs at up to 500 MHz. The ARM cores are 64-bit, and while the VideoCore is 32-bit, there is a new Memory Management Unit, which means it can access more memory than previous versions.
+The ARM cores are capable of running at up to 1.5 GHz, making the Raspberry Pi 4 about 50% faster than the Raspberry Pi 3B+. The new VideoCore VI 3D unit runs at up to 550 MHz. The GPU now has its own memory management unit (MMU), allowing it to make more efficient use of system memory.
```

Comment:
I've removed reference to the GPU being able to access more memory than previous versions, since the GPU on all Raspberry Pi computers can access the full range of system memory - the fact that previous GPUs were limited to 1GB is neither here nor there, given they only had 1GB or RAM anyway.

`More efficent use of system memory` refers to the memory being allocated dynamically from Linux via CMA, rather than being statically allocated via `gpu_mem=x` in `config.txt`.